bin/custom-kernel

```diff
@@ -173,11 +173,4 @@ case "$i" in
     esac
 done
 
-# Since we are doing a reboot, might as well also turn off mitigations and have a faster testbed
-echo "===> Disabling all mitigations"
```

Comment:
Why remove these?

Reply:
When I introduced them a while ago you highlighted that it wasn't a good idea to diverge that much from a real system.

Reply:
Ah that's why thanks